src/libraries/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -970,11 +970,10 @@ private async void CreateConnectionAsync()
             string hostname = _uri.Host;
```

Comment:
What if these throw? Or AsyncRequestCallback does?

Reply:
AsyncRequestCallback has a try/catch around its body.

Uri.Host/Port shouldn't throw at this point, but I don't mind moving them into the try.